### ISSUE TITLE
Merchandise doesn’t load on collections - page 2

### ISSUE THREAD

**Issue by mihaela-i** [mirceageorgescu/hop-hooligans#7]
Merchandise section is empty here:

https://www.hophooligans.ro/collections?page=2

Looking on Chrome - iPhone 6S- iOS12

**Comment by mirceageorgescu** [comment]
Yeah, it's just stupid. In the meantime I'll get rid of page 2 and the whole pagination thing.

**Comment by mihaela-i** [comment]
Pff classic “we’ll just remove it so we don’t have to fix” 🙄

**Comment by mirceageorgescu** [comment]
Exactly. Missed that?

**Comment by mihaela-i** [comment]
YES! 😁🎉
